etc/taskcluster/decision_task.py

```diff
@@ -529,6 +530,10 @@ def windows_nightly():
     )
 
 
+def webrender_nightly():
+    return linux_wpt_common(total_chunks=4, layout_2020=False, webrender_nightly=True)
```

Comment:
Nit: I’d prefer to keep the calls to `linux_wpt_common` closer to each other, so we see them all when changing for example the number of chunks.